phase1/gce/do

```diff
@@ -53,9 +56,18 @@ fetch_kubeconfig() {
     # succeeds, then stdout is tainted by ssh initialization output, so we echo
     # a marker to indicate the start of the file, and delete all lines up to and
     # including it.
-    if gcloud compute ssh --project "$PROJECT" --zone "$ZONE" "$MASTER" --command "echo STARTFILE; sudo cat /etc/kubernetes/admin.conf" > ${TMP_DIR}/kubeconfig.raw; then
+    if gcloud compute ssh --project "$PROJECT" --zone "$ZONE" "$SSHUSER$MASTER" --command "echo STARTFILE; sudo cat /etc/kubernetes/admin.conf" > ${TMP_DIR}/kubeconfig.raw; then
       sed '/^STARTFILE$/,$!d;/^STARTFILE$/d' ${TMP_DIR}/kubeconfig.raw > ${CLUSTER_NAME}/kubeconfig.json
       echo Successfully fetched kubeconfig.
+
+      if [[ -n "${KUBE_CONTEXT_NAME}" ]]; then
```

Comment:
Nitpick: can we aim for consistency in the style of these checks? You have two different `if`s to check for a non-empty string, and in the above case you use `if [[ "${VAR}" != "" ]]` but use `if [[ -n "${VAR}" ]]` here. I'm not sure which pattern is dominant in the repo right now, but we should standardize on one form.